### DISCUSSION TITLE
VagasBR - Agregador de vagas para Dev espalhadas pelo github.

### DISCUSSION THREAD

**Discussion by Ermesoml** [frontendbr/forum#1435] [Discussão]
Olá pessoal, estive no último mês trabalhando numa ferramenta para agregar as vagas espalhadas em diversos repositórios do github. Ele faz a busca em repositórios como frontendbr, backendbr, etc.

Deixo ela aqui para que a comunidade possa desfrutar e quem sabe contribuir. A stack é:
- Vue.js
- Node.js (Express)
- Mongodb

Está com deploy automático no heroku e netlify então todos os commits após aprovados serão já aplicados em produção.

Funcionamento: Um scheduler busca a cada hora as vagas contidas nos repositórios contidos em schedulerScripts/repositorio.js atualizando uma instância mongodb na atlas. Resolvi guardar as vagas para melhorar o controle sobre os dados e possibilitar a criação de relatórios no futuro.

Free and opensource!

https://vagasbr.netlify.com/#/

Espero que seja útil ;)

Front: https://github.com/Ermesoml/VagasBRPWA
Back: https://github.com/Ermesoml/VagasBRAPI

**Comment by felipefialho** [comment]
Muito bom @Ermesoml

**Comment by matpratta** [comment]
Poderia incluir uma sessão no estilo do [me-contrata](https://github.com/frontendbr/me-contrata), que listasse o pessoal buscando vagas para as empresas também!

**Comment by Ermesoml** [comment]
> Poderia incluir uma sessão no estilo do [me-contrata](https://github.com/frontendbr/me-contrata), que listasse o pessoal buscando vagas para as empresas também!

Cara, é uma ótima ideia. Farei essa página lá, deixar separado a parte das vagas e de perfis de profissionais. Abri uma issue pra isso

https://github.com/Ermesoml/VagasBRPWA/issues/1

**Comment by willycamargo** [comment]
Ficou muito bom, @Ermesoml ! Parabéns pela criatividade e iniciativa.

Um ótimo material de estudo para quem busca estudar começar com Node e Vue.js. Ta salvo aqui para passar como referência para alunos 👏 👏 👏.

**Comment by NicholasKuchiniski** [comment]
Boa, se alguém tiver interessado em algo parecido com isso mas feito em React, eu fiz um a algum tempo: https://github.com/NicholasKuchiniski/frontendbr-vagas / https://vagas.nicholaskuchiniski.dev/

**Comment by Ermesoml** [comment]
> Ficou muito bom, @Ermesoml ! Parabéns pela criatividade e iniciativa.
> 
> Um ótimo material de estudo para quem busca estudar começar com Node e Vue.js. Ta salvo aqui para passar como referência para alunos 👏 👏 👏.

Opa, valeu meu caro! E é exatamente o que pensei ao criar, mostrar como de fato podemos fazer algo "real" com essas tecnologias. Vejo muito material mostrando de modo separado, mas pra quem está começando o interessante é ver tudo junto funcionando.

**Comment by Ermesoml** [comment]
> Boa, se alguém tiver interessado em algo parecido com isso mas feito em React, eu fiz um a algum tempo: https://github.com/NicholasKuchiniski/frontendbr-vagas / https://vagas.nicholaskuchiniski.dev/

Bacana demais meu caro! Quero no futuro fazer uma versão em RN.
